README.md

```diff
@@ -0,0 +1,8 @@
+The signtool is an utility that can be used to issue sign/verify requests for a development or
+production mode.
```

Comment:
What is development mode?

What is production mode?

WHy would you do one over the other?